src/scenarios/weblarge2.0/test.py

```diff
@@ -0,0 +1,21 @@
+import os
+from shared.runner import TestTraits, Runner
+from shared import const
+
+SCENARIONAME = 'Web Large 2.0'
+EXENAME = 'weblarge20'
+
+def main():
+    traits = TestTraits(scenarioname=SCENARIONAME,
+                        exename=EXENAME,
+                        guiapp='false',
+                        workingdir='mvc',
+                        timeout='1000',  # increase timeout for the large project
```

Comment:
1000 seconds is a weird value here. Typically we do this sort of thing in multiples of 60 seconds. Maybe 900 or 1200 here? (Bonus points: something like `f'{60*15}'`, or `f'{MINUTE*15}'` and define `MINUTE` somewhere.)

Reply:
Thanks for the suggestion! Added the constant to ```const.py```